src/core/client/admin/components/UserHistoryDrawer/UsernameChangeAction.tsx

```diff
@@ -0,0 +1,38 @@
+import { Localized } from "fluent-react/compat";
+import React, { FunctionComponent } from "react";
+
+import styles from "./UsernameChangeAction.css";
+
+export interface UsernameChangeActionProps {
+  username: string;
+  prevUsername: string | null;
+}
+
+const SuspensionAction: FunctionComponent<UsernameChangeActionProps> = ({
+  username,
+  prevUsername,
+}) => {
+  return (
+    <div className={styles.usernameCell}>
+      <Localized id="moderate-user-drawer-username-change">
+        <div>Username change</div>
+      </Localized>
+      <div>
+        <Localized id="moderate-user-drawer-username-change-new">
+          <span className={styles.tableLight}>New: </span>
+        </Localized>
+        {` ${username}`}
+      </div>
+      {prevUsername && (
+        <div>
+          <Localized id="moderate-user-drawer-username-change-old">
+            <span className={styles.tableLight}>Old: </span>
+          </Localized>
+          {` ${prevUsername}`}
```

Comment:
I'd prefer:

```tsx
          <Localized id="moderate-user-drawer-username-change-old">
            <span className={styles.tableLight}>Old:</span>
          </Localized>{" "}
          {prevUsername}
```

Instead as it reads better than a string template. Also note that trailing spaces cannot be included in translations, and should not be depended on, (hence me removing the ` ` from `<span className={styles.tableLight}>Old: </span>`).